iib/workers/tasks/build.py

```diff
@@ -1112,9 +1112,10 @@ def handle_rm_request(
             _build_image(temp_dir, 'index.Dockerfile', request_id, arch)
             _push_image(request_id, arch)
 
-    _verify_index_image(
-        prebuild_info['from_index_resolved'], from_index, overwrite_from_index_token
-    )
+    if overwrite_from_index:
+        _verify_index_image(
```

Comment:
Maybe in the future, but it would be great if this was done as part of the `_overwrite_from_index` function.

Reply:
Done